src/ImageViewing.tsx

```diff
@@ -36,6 +36,7 @@ type Props = {
   doubleTapToZoomEnabled?: boolean;
   HeaderComponent?: ComponentType<{ imageIndex: number }>;
   FooterComponent?: ComponentType<{ imageIndex: number }>;
+  wrapperStyle?: Object;
```

Comment:
It's better to do strict typing using `wrapperStyle?: StyleProp<ViewStyle>` there `StyleProp` and `ViewStyle` are imported types from `react-native`.